commands/members/add.js

```diff
@@ -5,16 +5,25 @@ let co     = require('co');
 let extend = require('util')._extend;
 
 function* run (context, heroku) {
+  const FREE_TEAM_LIMIT = 5;
   let org   = context.org;
   let email = context.args.email;
   let role  = context.flags.role;
+  let members = yield heroku.get(`/organizations/${org}/members`);
+  let features = yield heroku.get('/account/features');
+  let orgCreationFeature = features.find(function(feature) {
```

Comment:
Features is an array of objects like ['name' : 'standard-org-creation', ['name': 'org-access-controls']. So I couldn't directly look for the string. I'll check if includes works with comparing objects, though..